composer.json

```diff
@@ -39,7 +39,8 @@
     "require-dev": {
         "horde/test": "^3 || dev-FRAMEWORK_6_0",
         "horde/cli": "^3 || dev-FRAMEWORK_6_0",
-        "horde/scribe": "^3 || dev-FRAMEWORK_6_0"
+        "horde/scribe": "^3 || dev-FRAMEWORK_6_0",
+        "phpunit/phpunit": "10.0.x-dev"
```

Comment:
was this intentional or by accident?
The horde/test requirement should already install phpunit9. Do we need phpunit10 for anything?

Reply:
Neither should. A package should require-dev on horde/test and make no assumption if phpunit is installed to the project or called using an external phar.

Reply:
@ralflang @midahp falsely added, correct on in usr/local/bin in use